h5c/vic/src/vic-webapp/src/app/vch-view/vch-view.component.ts

```diff
@@ -225,6 +231,50 @@ export class VicVchViewComponent implements OnInit, OnDestroy {
     this.vmViewService.getContainersData({}); // TODO: filter containers for the vch
   }
 
+   /**
+   * Download Certificate for VCH
+   */
+  downloadVchCert(vch: VirtualContainerHost) {
+    this.isDgLoading = true;
+    Observable.combineLatest(
+      this.createWzService.getVicApplianceIp(),
+      this.createWzService.acquireCloneTicket()
+    ).catch(err => {
+      return Observable.throw(err);
+    }).subscribe(([serviceHost, cloneTicket]) => {
+      const vchId = vch.id.split(':')[3];
```

Comment:
Could we use `vch.name` inststead of `vch.id` for the pem file name? It would be easier for users to realize which pem file is for which VCH. MOID is rather for internal referencing purposes.